### ISSUE TITLE
[yggtorrent] Exception (yggtorrent): FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Cloudflare Error: Execution context was destroyed, most likely because of a navigation.: FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Cloudflare Error: Execution context was destroyed, most likely because of a navigation. (Test)

### ISSUE THREAD

**Issue by navycrow** [FlareSolverr/FlareSolverr#228]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

### How to enable debug and html traces

[Follow the instructions from this wiki page](https://github.com/FlareSolverr/FlareSolverr/wiki/How-to-enable-debug-and-html-trace)

### Environment

* **FlareSolverr version**:
* **Last working FlareSolverr version**:
* **Operating system**:
* **Are you using Docker**: [yes/no]
* **FlareSolverr User-Agent (see log traces or / endpoint)**:
* **Are you using a proxy or VPN?** [yes/no]
* **Are you using Captcha Solver:** [yes/no]
* **If using captcha solver, which one:**
* **URL to test this issue:**

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]

### Logged Error Messages

[Place any relevant error messages you noticed from the logs here.]

[Make sure you attach the full logs with your personal information removed in case we need more information]

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by ilike2burnthing** [comment]
If you want help, complete the information requested.

**Comment by thefyfy** [comment]
Hi,

Same here. More informations below : 

### Environment

* **FlareSolverr version**: v1.2.3
* **Last working FlareSolverr version**: 🤷‍♂️
* **Operating system**: Synology DSM 6.2.4-25556 Update 2
* **Are you using Docker**: yes
* **FlareSolverr User-Agent (see log traces or / endpoint)**:
* **Are you using a proxy or VPN?** no
* **Are you using Captcha Solver:** not sure
* **If using captcha solver, which one:**
* **URL to test this issue:** https://www3.yggtorrent.re/engine/search?category=all&name=&description=&file=&uploader=&sub_category=&do=search&order=desc&sort=publish_date

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]
Run a connection test from jackett to YGGTorrent

### Logged Error Messages

2021-12-07T08:45:40.135Z INFO REQ-4 Incoming request: POST /v1
2021-12-07T08:45:40.135Z INFO REQ-4 Params: {"maxTimeout":55000,"cmd":"request.get","url":"https://www3.yggtorrent.re/engine/search?category=all&name=&description=&file=&uploader=&sub_category=&do=search&order=desc&sort=publish_date"}
2021-12-07T08:45:41.085Z INFO REQ-4 Cloudflare detected
2021-12-07T08:45:46.313Z INFO REQ-4 Successful response in 6.178 s
2021-12-07T08:45:46.313Z INFO REQ-4 Cloudflare Error: Execution context was destroyed, most likely because of a navigation.

### Screenshots

Jackett screenshot : 
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/1229018/144997886-5f5573eb-09a1-475d-ad1d-7136feffef38.png">

If I can help with others informations, let me know.

Thanks

**Comment by thefyfy** [comment]
Missing parameter for flaresolverr in my watchtower config...🤦‍♂️
Sorry about that and thanks, it works like a charm !

**Comment by ilike2burnthing** [comment]
@navycrow I would assume you need to update as well. Reopen the issue with the requested information if you continue to have problems once updated.

**Comment by DaGreenX** [comment]
Hey guys, I have similar config and issue. 
little question for a newbe, how can I update FlareSolver from 1.2.3 to v2 ?

**Comment by ilike2burnthing** [comment]
You're using Docker, so pull a new image and redeploy.

**Comment by DaGreenX** [comment]
ok done.

not sure for the parameters, because I have an issue again.

This is my parameters and logs.

<img width="625" alt="image" src="https://user-images.githubusercontent.com/32598185/145357538-daca179c-434a-4ff7-b89e-46886c3be8d0.png">

<img width="636" alt="image" src="https://user-images.githubusercontent.com/32598185/145357596-85b30048-b2a1-4ee0-9b64-9d33400a9c5a.png">



2021-12-09 08:01:15 | stdout | 2021-12-09T08:01:15+00:00 ERROR REQ-2 Unexpected error: Error: Protocol error (Network.getCookies): Session closed. Most likely the page has been closed.
-- | -- | --
2021-12-09 08:01:14 | stdout | 2021-12-09T08:01:14+00:00 INFO REQ-2 Response in 60.649 s
2021-12-09 08:01:14 | stdout | 2021-12-09T08:01:14+00:00 ERROR REQ-2 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
2021-12-09 08:00:22 | stdout | 2021-12-09T08:00:22+00:00 INFO REQ-2 Cloudflare detected
2021-12-09 08:00:14 | stdout | 2021-12-09T08:00:14+00:00 INFO REQ-2 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://www3.yggtorrent.re/"}
2021-12-09 08:00:09 | stdout | 2021-12-09T08:00:09+00:00 INFO REQ-1 Response in 9.299 s
2021-12-09 08:00:09 | stdout | 2021-12-09T08:00:09+00:00 INFO REQ-1 Cloudflare Error: Cloudflare has blocked this request. Probably your IP is banned for this site, check in your web browser.
2021-12-09 08:00:09 | stdout | 2021-12-09T08:00:09+00:00 INFO REQ-1 Cloudflare detected
2021-12-09 08:00:00 | stdout | 2021-12-09T08:00:00+00:00 INFO REQ-1 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://www3.yggtorrent.re/"}
2021-12-09 07:58:42 | stdout | 2021-12-09T07:58:42+00:00 INFO Listening on http://0.0.0.0:8191
2021-12-09 07:58:42 | stdout | 2021-12-09T07:58:42+00:00 INFO Test successful
2021-12-09 07:58:42 | stdout | 2021-12-09T07:58:42+00:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0
2021-12-09 07:58:32 | stdout | 2021-12-09T07:58:32+00:00 INFO Testing web browser installation...
2021-12-09 07:58:32 | stdout | 2021-12-09T07:58:32+00:00 INFO FlareSolverr v2.0.2
2021-12-09 07:58:04 | stdout | 2021-12-09T07:58:04+00:00 INFO REQ-2 Process interrupted
2021-12-09 07:54:48 | stdout | 2021-12-09T07:54:48+00:00 ERROR REQ-2 Unexpected error: Error: Protocol error (Network.getCookies): Session closed. Most likely the page has been closed.
2021-12-09 07:54:47 | stdout | 2021-12-09T07:54:47+00:00 INFO REQ-2 Response in 59.527 s
2021-12-09 07:54:47 | stdout | 2021-12-09T07:54:47+00:00 ERROR REQ-2 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
2021-12-09 07:53:55 | stdout | 2021-12-09T07:53:55+00:00 INFO REQ-2 Cloudflare detected
2021-12-09 07:53:48 | stdout | 2021-12-09T07:53:48+00:00 INFO REQ-2 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://www3.yggtorrent.re/"}
2021-12-09 07:53:42 | stdout | 2021-12-09T07:53:42+00:00 INFO REQ-1 Response in 8.671 s
2021-12-09 07:53:42 | stdout | 2021-12-09T07:53:42+00:00 INFO REQ-1 Cloudflare Error: Cloudflare has blocked this request. Probably your IP is banned for this site, check in your web browser.
2021-12-09 07:53:42 | stdout | 2021-12-09T07:53:42+00:00 INFO REQ-1 Cloudflare detected
2021-12-09 07:53:34 | stdout | 2021-12-09T07:53:34+00:00 INFO REQ-1 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://www3.yggtorrent.re/"}
2021-12-09 07:53:28 | stdout | 2021-12-09T07:53:28+00:00 INFO Listening on http://0.0.0.0:8191
2021-12-09 07:53:28 | stdout | 2021-12-09T07:53:28+00:00 INFO Test successful
2021-12-09 07:53:28 | stdout | 2021-12-09T07:53:28+00:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0
2021-12-09 07:53:19 | stdout | 2021-12-09T07:53:19+00:00 INFO Testing web browser installation...
2021-12-09 07:53:19 | stdout | 2021-12-09T07:53:19+00:00 INFO FlareSolverr v2.0.2
2021-12-09 07:50:36 | stdout | 2021-12-09T07:50:36+00:00 INFO Process interrupted
2021-12-09 07:49:19 | stdout | 2021-12-09T07:49:19+00:00 INFO Listening on http://0.0.0.0:8191
2021-12-09 07:49:19 | stdout | 2021-12-09T07:49:19+00:00 INFO Test successful
2021-12-09 07:49:19 | stdout | 2021-12-09T07:49:19+00:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0
2021-12-09 07:49:10 | stdout | 2021-12-09T07:49:10+00:00 INFO Testing web browser installation...
2021-12-09 07:49:10 | stdout | 2021-12-09T07:49:10+00:00 INFO FlareSolverr v2.0.2
2021-12-09 07:44:24 | stdout | 2021-12-09T07:44:24+00:00 INFO Process interrupted
2021-12-09 07:44:20 | stdout | 2021-12-09T07:44:20+00:00 INFO Listening on http://0.0.0.0:8191
2021-12-09 07:44:20 | stdout | 2021-12-09T07:44:20+00:00 INFO Test successful
2021-12-09 07:44:20 | stdout | 2021-12-09T07:44:20+00:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0
2021-12-09 07:44:06 | stdout | 2021-12-09T07:44:06+00:00 INFO Testing web browser installation...
2021-12-09 07:44:06 | stdout | 2021-12-09T07:44:06+00:00 INFO FlareSolverr v2.0.2

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/234

TL;DR - switch to yggcookie or just keep trying yggtorrent again and again and again
